src/Operations.jl

```diff
@@ -937,8 +937,8 @@ function build_versions(ctx::Context, uuids::Vector{UUID}; might_need_to_resolve
                 success(pipeline(cmd, stdout=log, stderr=log))
             end
             if !ok
-                @error("Error building `$name`; see log file for further info")
-                build_succeeded = false
+                log_content = read(log_file, String)
+                @error("Error building `$name`: \n$log_content")
```

Comment:
I would only print out the last X lines.  In BinaryBuilder, we print out the last 100 lines of the log if things go south. These logs can get pretty heavy, if someone is e.g. building TensorFlow from source, you don't want to spam the console with hundreds of thousands of lines of text; but you do want to store it so the savvy user can look into it.  I would still print out the location of the logfile and also tail it out to the user.

Reply:
Alright, makes sense.